src/jit/flowgraph.cpp

```diff
@@ -3955,7 +3955,10 @@ bool Compiler::fgCreateGCPoll(GCPollType pollType, BasicBlock* block)
         //  2) Add a GC_CALL node to Poll.
         GenTreeCall* call = gtNewHelperCallNode(CORINFO_HELP_POLL_GC, TYP_VOID);
 #ifdef LEGACY_BACKEND
+#ifndef _TARGET_ARM_
+        // For more details see regalloc.cpp@4958, Compiler::rpPredictTreeRegUse
```

Comment:
I don't think a comment should refer to a line number in another file; they are too likely to change.
Also, I didn't find anything at that line number to elucidate what's going on here.
I presume it's the case that the ARM PollGC helper doesn't preserve all the registers. That would seem to be an adequate comment here.
Also, this may not be all that relevant if we are planning to move to RyuJIT for ARM, but I'm wondering if there should be a smaller kill set for this helper call than the standard call kill set.
I think that it would be good for @briansull to look at this as well, since he's more familiar with regalloc.cpp.

Reply:
Changed the comment.